### DISCUSSION TITLE
Appear Offline

### DISCUSSION THREAD

**Discussion by mmdevcodes** [ppy/osu#17440] [Q&A] [answered]
Is it possible to appear offline in lazer?

**Comment by smoogipoo** [comment, marked as answer]
There's an option for it in the user menu, but there's no support online states yet so it doesn't do anything.

**Reply by mmdevcodes** [reply]
Ah I see. Is there an open issue for implementing that or a roadmap for when that will be added?

**Reply by peppy** [reply]
Not yet. You can logout for now.

**Reply by frenzibyte** [reply]
There *is* an issue thread open for it, https://github.com/ppy/osu/issues/12635.
